RELEASE_NOTES.md

```diff
@@ -1,5 +1,152 @@
 # Release Notes
 
+## 2.2.0
+### Features
+- Experimental support for shortname aliasing has been added. This is not enabled by default, but can be turned on by setting the environment variable `CONTAINERS_SHORT_NAME_ALIASING` to `on`.
+- The `podman generate kube` command now features support for exporting container's memory and CPU limits ([#7855](https://github.com/containers/podman/issues/7855)).
+- The `podman play kube` command now features support for setting CPU and Memory limits for containers ([#7742](https://github.com/containers/podman/issues/7742)).
+- The `podman play kube` command now supports Kubernetes configmaps via the `--configmap` flag ([#7567](https://github.com/containers/podman/issues/7567)).
+- The `podman play kube` command now supports a `--log-driver` option to set the log driver for created containers.
+- The `podman run` command now supports the `--network-alias` flag to set network aliases (additional names the container can be accessed at from other containers via DNS if the `dnsname` CNI plugin is in use).
+- The `podman network create` command now supports the `--ipv6` flag to enable dual-stack IPv6 networking for created networks ([#7302](https://github.com/containers/podman/issues/7302)).
+- The `podman inspect` command can now inspect pods, networks, and volumes, in addition to containers and images ([#6757](https://github.com/containers/podman/issues/6757)).
+- The `--mount` option for `podman run` and `podman create` now supports a new type, `image`, to mount the contents of an image into the container at a given location.
+- The Bash and ZSH completions have been completely reworked and have received significant enhancements!
```

Comment:
We now also have completion support for fish and podman-remote. I believe this is more interesting for users and packagers.